scripts/zones/Temenos/mobs/Airi.lua

```diff
@@ -2,24 +2,26 @@
 -- Area: Temenos Central Floor
 --  Mob: Airi
 -----------------------------------
-require("scripts/globals/limbus");
------------------------------------
+local ID = require("scripts/zones/Temenos/IDs")
 
-function onMobEngaged(mob,target)
-  if (IsMobDead(16929047)==true) then
-     mob:addStatusEffect(tpz.effect.REGAIN,7,3,0);
-     mob:addStatusEffect(tpz.effect.REGEN,50,3,0);
-  end
-end;
+function onMobEngaged(mob, target)
+    if GetMobByID(ID.mob.TEMENOS_C_MOB[1]+1):isDead() then
+        mob:addStatusEffect(tpz.effect.REGAIN, 7, 3, 0)
+        mob:addStatusEffect(tpz.effect.REGEN, 50, 3, 0)
```

Comment:
Usual "this was already here, but...." disclaimer: What happens if a party decides to engage one, hold it, and then kill the linked mob to try to get around the "strength increasing" effect?

(Or a wipe after killing the linked mob.)

We may want to move the "gets stronger" logic to the onDeath of the linked mob.

Reply:
Yeah that sounds better. I think it's supposed to force a textless 2hr animation too, not sure how to do that.

Reply:
> supposed to force a textless 2hr animation too, not sure how to do that.

@59blargedy Time for (Vulture 2)? 😉

Reply:
i think substitute (id 307) is more commonly used for 2 hour 
cat 11 id 307 animation 439 message 0
(the vulture 2 and 3 thing from sea are weird?!?!)